manager/controllers/app/policy_compiler_interface.go

```diff
@@ -38,8 +38,16 @@ func ConstructApplicationContext(datasetID string, input *app.FybrikApplication,
 // LookupPolicyDecisions provides a list of governance actions for the given dataset and the given operation
 func LookupPolicyDecisions(datasetID string, policyManager connectors.PolicyManager, input *app.FybrikApplication, op *pb.AccessOperation) ([]*pb.EnforcementAction, error) {
 	// call external policy manager to get governance instructions for this operation
+	log.Println("AccessOperation before calling ConstructApplicationContext in LookupPolicyDecisions", op)
 	appContext := ConstructApplicationContext(datasetID, input, op)
-	pcresponse, err := policyManager.GetPoliciesDecisions(context.Background(), appContext)
+	log.Println("appContext before calling GetPoliciesDecisions in LookupPolicyDecisions", appContext)
+	openapiReq, creds, _ := connectors.ConvertGrpcReqToOpenAPIReq(appContext)
+	log.Println("openapiReq after calling ConvertGrpcReqToOpenAPIReq in LookupPolicyDecisions", openapiReq)
+	openapiResp, err := policyManager.GetPoliciesDecisions(openapiReq, creds)
+	log.Println("openapiresp after GetPoliciesDecisions in LookupPolicyDecisions", openapiResp)
+	pcresponse, _ := connectors.ConvertOpenAPIRespToGrpcResp(openapiResp, datasetID, op)
+	log.Println("pcresponse after GetPoliciesDecisions in LookupPolicyDecisions", pcresponse)
```

Comment:
Too verbose, do we really need all of these?

Reply:
Modified the logs to remove unnecessary logs and modified log messages also. Please review. Thanks.